_commands/tools/mpage.md

```diff
@@ -0,0 +1,59 @@
+---
+---
+
+mpage
+---
+
+`mpage` is used to print pages to a postcript printer. It is useful at condensing output, and can condense up to 8 pages onto a single piece of paper.
+
+~~~ bash
+$ mpage [options] <files...>
+~~~
+
+<!--more-->
+
+###Useful Options / Examples
+
```

Comment:
Put a space after `###` and "Useful ...", it doesn't render properly otherwise.
![screen shot 2016-03-27 at 5 10 19 pm](https://cloud.githubusercontent.com/assets/7095187/14067902/e6450a8c-f43e-11e5-9e7f-af657eb0dfb2.png)